src/collective/exportimport/serializer.py

```diff
@@ -326,9 +326,10 @@ def __call__(self):
             return json_compatible(result)
 
     def get_at_blob_path(obj):
-        full_path = obj.getBlob().committed()
-        if not os.path.exists(full_path):
-            return
```

Comment:
My thought was: if no blob exists at a path, then it is useless to return this path: importing it would fail because there is nothing.

When I follow the code of the `committed` method in `ZODB/blob.py`, calling `loadBlob`, I see that you actually get a POSKeyError when no file exists on that path. So in hindsight, my file existence check would either pass successfully, or not get called at all...
In other words: it is safe to remove.